src/interface/src/app/map/map.component.ts

```diff
@@ -77,10 +77,10 @@ export class MapComponent implements AfterViewInit, OnDestroy {
     );
   }
 
-  static openStreetMapsTiles() {
-    return L.tileLayer('https://tile.openstreetmap.org/{z}/{x}/{y}.png', {
+  static stadiaAlidadeTiles() {
+    return L.tileLayer('https://tiles.stadiamaps.com/tiles/alidade_smooth/{z}/{x}/{y}{r}.png', {
       maxZoom: 19,
-      attribution: '© OpenStreetMap',
+      attribution: '&copy; <a href="https://stadiamaps.com/">Stadia Maps</a>, &copy; <a href="https://openmaptiles.org/">OpenMapTiles</a> &copy; <a href="http://openstreetmap.org">OpenStreetMap</a> contributors',
```

Comment:
Should these links open in a new tab? (Add target="_blank)

Can also add rel="noreferrer" to prevent passing referrer information to the target website by removing the referral info from the HTTP header.